httpx/_dispatch/wsgi.py

```diff
@@ -88,6 +97,9 @@ def start_response(
             seen_exc_info = exc_info
 
         result = self.app(environ, start_response)
+        # This is needed because the status returned by start_response
+        # shouldn't be used until the first non-empty chunk has been served.
+        result = _get_non_empty_chunk(result)
```

Comment:
Woops 😄  (Forgot to add this suggestion to my previous review!)

```suggestion
        result = _skip_leading_empty_chunks(result)
```